tools/sklearn/keras_train_and_eval.xml

```diff
@@ -19,30 +19,27 @@
             --inputs '$inputs'
             --estimator '$experiment_schemes.infile_estimator'
             #if $input_options.selected_input == 'seq_fasta'
-            --fasta_path '$input_options.fasta_path'
+                --fasta_path '$input_options.fasta_path'
             #elif $input_options.selected_input == 'refseq_and_interval'
-            --ref_seq "`pwd`/${ref_genome_file.element_identifier}"
-            --interval '$input_options.interval_file'
-            --targets "`pwd`/${target_file.element_identifier}.gz"
+                --ref_seq "`pwd`/${ref_genome_file.element_identifier}"
+                --interval '$input_options.interval_file'
+                --targets "`pwd`/${target_file.element_identifier}.gz"
             #else
-            --infile1 '$input_options.infile1'
+                --infile1 '$input_options.infile1'
             #end if
-            --infile2 '$input_options.infile2'
-            --outfile_result "`pwd`/tmp_outfile_result"
+                --infile2 '$input_options.infile2'
+                --outfile_result '$outfile_result'
             #if $save and 'save_estimator' in str($save)
-            --outfile_object '$outfile_object'
-            --outfile_weights '$outfile_weights'
+                --outfile_object '$outfile_object'
+                --outfile_weights '$outfile_weights'
             #end if
             #if $save and 'save_prediction' in str($save)
-            --outfile_y_true '$outfile_y_true'
-            --outfile_y_preds '$outfile_y_preds'
+                --outfile_y_true '$outfile_y_true'
+                --outfile_y_preds '$outfile_y_preds'
             #end if
             #if $experiment_schemes.test_split.split_algos.shuffle == 'group'
-            --groups '$experiment_schemes.test_split.split_algos.groups_selector.infile_g'
+                --groups '$experiment_schemes.test_split.split_algos.groups_selector.infile_g'
             #end if
-            >'$outfile_result' && cp '$outfile_result' "`pwd`/../tool_stdout"
```

Comment:
@kxk302 this is the error. This is really not good and was not caught without the container tests. So its good we have these more stringent tests.